react-app/src/components/UI/RecursiveComponent.tsx

```diff
@@ -0,0 +1,26 @@
+import { RecursiveProps } from "../../models/RecursiveProps.ts";
+import styles from '../../styles/RecursiveComponent.module.css';
+
+const getColor = (depth : number): string => {
+  const hue = 190 + (depth * 3);
+  const saturation = 96+ (depth * 3);
```

Comment:
In fact this should be secured against overflow over 255 value. If this doesn't crash, it does not mean it's valid.

Reply:
or maybe you wish it to start from black when it overflows, then it's ok

Reply:
changed.